bin/validateSchema.js

```diff
@@ -22,25 +26,81 @@ ajv.addFormat('date-time', {
   validate: (data) => typeof data === 'string' && /\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(.\d{1,3})?Z/.test(data),
 });
 
-function getJsonSchema() {
+function getJsonSchemaValidator() {
   const apiDefinition = yaml.load(fs.readFileSync('./schemes/fingerprint-server-api.yaml'));
-  return convertOpenApiToJsonSchema(apiDefinition, '#/definitions/Response');
+  const schema = convertOpenApiToJsonSchema(apiDefinition, '#/definitions/Response');
+  return ajv.compile(schema);
+}
+
+function getJsonDataMockObjects() {
+  const mocks = [
+    {
+      name: 'Limit 1',
+      path: './examples/visits_limit_1.json',
+    },
+    {
+      name: 'Limit 500',
+      path: './examples/visits_limit_500.json',
+    },
+  ];
+
+  return mocks.map(({ name, path }) => ({
+    name: `${name} mock`,
+    jsonData: JSON.parse(fs.readFileSync(path).toString()),
```

Comment:
Maybe we could just do:
```suggestion
    jsonData: require(path),
```
?

Reply:
Not for ES Module, can replace for it, but not sure it is simpler (a lot of things became async plus to the snippet):

```suggestion
     jsonData: (
        await import(path, {
          assert: {
            type: 'json',
          },
        })
      ).default,
```

Reply:
Ah, good point, I've forgot that this is not a ES Module!